torch/legacy/nn/MarginRankingCriterion.py

```diff
@@ -60,7 +60,7 @@ def updateGradInput(self, input, y):
             self.mask = self.mask.resize_as_(input[0]).copy_(dist)
             mask = self.mask
 
-            torch.ge(dist, 0, out=mask)
+            torch.ge_t(dist, 0, out=mask)
```

Comment:
I'd rather not get caught up on this minor part of the library and let it delay making TensorLib usable, so we can just leave the python behavior unchanged and not expose the real-returning variant in C++ at all.